### ISSUE TITLE
ESM error

### ISSUE THREAD

**Issue by kaname-png** [Dexare/Voltare#20]
I want to use this package but I run into the problem of:

```ts
..\node_modules\voltare\lib\client\index.js:26
const Revolt = __importStar(require("revolt.js"));
                            ^

Error [ERR_REQUIRE_ESM]: require() of ES Module ..\node_modules\revolt.js\dist\index.js from ..\node_modules\voltare\lib\client\index.js not supported.
Instead change the require of ..\node_modules\revolt.js\dist\index.js in ..\node_modules\voltare\lib\client\index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (..\node_modules\voltare\lib\client\index.js:26:29)
    at Object.<anonymous> (..\node_modules\voltare\lib\index.js:29:34) {
  code: 'ERR_REQUIRE_ESM'
}
```

I have added the property "type": "module" to the package.json file but it doesn't work either, I have also changed the target of the tsconfig.json file and it doesn't work either.

**Comment by Snazzah** [comment]
In terms of *not* using ESM, it's probably better if I do a major release to move to ESM for this package.

**Comment by Spaxly** [comment]
> In terms of _not_ using ESM, it's probably better if I do a major release to move to ESM for this package.

Any ETA on this?

**Comment by Snazzah** [comment]
Released [v1.0.0](https://github.com/Dexare/Voltare/releases/tag/v1.0.0).
